### DISCUSSION TITLE
Firebase Crashlytics non-fatal issue in main.jsbundle

### DISCUSSION THREAD

**Discussion by TommyLeong** [firebase/firebase-js-sdk#4371] [Archive] [answered]
Not sure if I'm opening this discussion at the right repository group, otherwise I appreciate your help to direct me to the right group. This is a discussion related to Firebase Crashlytics Report for React Native (JS) application.

### What's it about ###

Hello all, I've been receiving the following in my email recently. However, the same crash report can't be found on the portal itself when I go into my app's Crashlytics details.

It only navigates me to this specific crash details while I tap on the `LEARN MORE` button in the email. Otherwise, I don't get to find it. Any idea what's this about?

I have the issue 
[posted on Stackoverflow](https://stackoverflow.com/questions/65939473/firebase-crashlytics-non-fatal-issue-in-main-jsbundle)
[posted on RNFirebase Github Discussion](https://github.com/invertase/react-native-firebase/discussions/4833#discussioncomment-320889)

<img width="353" alt="Screenshot 2021-01-29 at 2 24 23 PM" src="https://user-images.githubusercontent.com/9990179/106239503-b12d5f80-623d-11eb-9d47-7e4207fa9762.png">

**Comment by samtstern** [comment, marked as answer]
@TommyLeong this is the repository for the Firebase JS SDK so it's not the right place to discuss issues about Crashlytics (not supported on the web) or React Native (https://github.com/invertase/react-native-firebase).

**Reply by TommyLeong** [reply]
Hi @samtstern, would you be able to suggest me the right repo for discussing this issue? As I don't spot any specific crashlytics repo for me to put in this ques. 

[firebase-ios-sdk](https://github.com/firebase/firebase-ios-sdk/tree/master/Crashlytics) is the only closest I found, but I'm not using iOS sdk.

**Reply by samtstern** [reply]
Since your issue is with the backend and the email alerts and doesn't seem like an SDK issue, try Firebase Support:
https://firebase.google.com/support
